runtime/vm/j9vm.tdf

```diff
@@ -927,3 +927,5 @@ TraceExit=Trc_VM_allocateThunkHeap_Exit NoEnv Overhead=1 Level=3 Template="Exit
 TraceEvent=Trc_VM_allocateThunkHeap_allocate_thunk_heap_node_failed NoEnv Overhead=1 Level=3 Template="allocateThunkHeap - Failed to allocate memory for the thunk heap node"
 
 TraceExit=Trc_VM_criu_initHooks_Exit Overhead=1 Level=5 Template="initializeCriuHooks - checkpointState.hookRecords (%p), classIterationRestoreHookRecords (%p), delayedLockingOperationsRecords (%p)"
+
+TraceException=Trc_VM_checkVisibility_Failed1 Overhead=1 Level=1 Template="checkVisibility from %p (%.*s) to %p (%.*s) modifiers=%zx lookupOptions=%zx failed"
```

Comment:
Please reuse the tracepoint name `Trc_VM_checkVisibility_Failed`.

Reply:
So it means we can have the same name for two trace points. One obsolete and one in use ?

Reply:
The obsolete tracepoint is not used in builds other than to satisfy references in old binary trace logs.